### ISSUE TITLE
unable to change font-size-base and/or font.global.size

### ISSUE THREAD

**Issue by jodumont** [theme-next/hexo-theme-next#629]
Hi;

I'm using `source/_data/variables.styl` to make variation inside **Gemini** scheme but one change doesn't apply and I wonder why `$font-size-base = 16px` don't works.

this is my variables.styl
```

// Variables of Gemini scheme
    
// corpo color
$blue    = #192b33
$green   = #8fb259
$orange  = #ff7f00
$white   = #f2f2f2
$black   = #2f2f2f

// custom base
$font-size-base = 18px
$whitesmoke = $white
$black-deep = $blue
$blue-deep = $blue

// Settings for some of the most global styles.
$body-bg-color = $white

// Borders
$box-shadow-inner = 0 2px 2px 0 rgba(47,47,47,.12), 0 3px 1px -2px rgba(47,47,47,.06), 0 1px 5px 0 rgba(47,47,47,.12)
$box-shadow = 0 2px 2px 0 rgba(47,47,47,.12), 0 3px 1px -2px rgba(47,47,47,.06), 0 1px 5px 0 rgba(47,47,47,.12), 0 -1px .5px 0 rgba(47,47,47,.09)
$border-radius-inner = 0 0 3px 3px;
$border-radius = 3px;

// Logo
.custom-logo-image {max-width: 288px;}

// Sidebar
$subtitle-font-size = 14px
$site-description-align = left
.site-subtitle {margin-top: 0!important;}

// Footer
.footer {
    color: $white!important
    background-color: $blue
  a {color: $orange}
  a:hover {color: $green}
}

// Color
.green {color: $green}
.blue {color: $blue}
```

base on what I found in #245 
I also tried by adding this in my `_config.yml and source/data/next.yml`
without success

```
font:
  global:
    size: 18px
```

**Comment by stevenjoezhang** [comment]
Setting
```
font:
  global:
    size: 18px
```
should work fine.
Have you tried `hexo clean && hexo g && hexo s` ?

**Comment by jodumont** [comment]
hum!

I was using `hexo clean && hexo serve`
and now even with `hexo clean && hexo g && hexo s`

with in `_config.yml` or `source/data/next.yml`

```
font:
  global:
    size: 18px
```

Gemini Scheme don't respect it

Just to be sure
1. I'm suppose to have my main font at 18px when I'm using hexo server ?
2. I don't have to declare anything else in the font ?

_aka I don't have enable like this_

```
font:
  enable: false
  global:
    size: 18px
```

**Comment by stevenjoezhang** [comment]
Can you provide an url to a site contains this bug? Or can you confirm if there's the same bug in other schemes?

**Comment by jodumont** [comment]
I'll try another schemes and let's you know :)

**Comment by jodumont** [comment]
@stevenjoezhang I also tried on a new hexo/next project with Mist and it didn't works
but don't worry it's probably something I didn't understood so i'll figured out later on my own
;)

**Comment by stevenjoezhang** [comment]
@jodumont Ahh, I made a mistake. It should not be
```yaml
font:
  global:
    size: 18px
```
But
```yaml
font:
  global:
    size: 18
```
You can try again

**Comment by stevenjoezhang** [comment]
Fixed in #1005, #1006
